travis/travis_install_nightly

```diff
@@ -211,7 +211,22 @@ if [ "x${WKHTMLTOPDF_VERSION}" == "x" ] ; then
     WKHTMLTOPDF_VERSION="0.12.3"
 fi
 echo "Installing the version of webkit ${WKHTMLTOPDF_VERSION}"
-(cd ${HOME}/maintainer-quality-tools/travis/ && wget -qO- -t 1 --timeout=240 https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/${WKHTMLTOPDF_VERSION}/wkhtmltox-${WKHTMLTOPDF_VERSION}_linux-generic-amd64.tar.xz | tar -xJ --strip-components=2 wkhtmltox/bin/wkhtmltopdf)
+
+if [[ ${WKHTMLTOPDF_VERSION} == "0.12.3" ]]
```

Comment:
@moylop260 This issue https://github.com/Vauxoo/maintainer-quality-tools/issues/227 was open for add support the version 0.12.1. Now also is need add support the version 0.12.4 ?